packages/hello-world/README.md

```diff
@@ -1 +1,26 @@
 # HelloWorld
+
+Mineral UI `HelloWorld` component. Find out more about [Mineral UI](https://github.com/mineral-ui/mineral-ui).
+
+## Installation
+
+```sh
+npm install --save @mineral-ui/hello @mineral-ui/world @mineral-ui/hello-world
+```
+
+## Usage
+
+```jsx
+import React, { Component } from 'react';
+import HelloWorld from '@mineral-ui/hello-world';
+
+export default class MyComponent extends Component {
+  render() {
+    return (
+      <div>
```

Comment:
I think so since it makes it a more realistic example. I doubt people will be making their own components that simply render ours. I guess they could want to create a component that just changes the prop signature... but that seems far fetched to me.